Exercise_1.py

```diff
@@ -1,19 +1,33 @@
 class myStack:
-     def __init__(self):
-         
-     def isEmpty(self):
-         
-     def push(self, item):
-         
-     def pop(self):
-        
-        
-     def peek(self):
-        
-     def size(self):
-         
-     def show(self):
-         
+    def __init__(self):
```

Comment:
This implementation does not fulfill the requirements for the assignment. Please use the top variable to keep track of top of stack.